mainnet-mirror/genesis.json

```diff
@@ -290,7 +303,7 @@
     },
     "version": {}
   },
-  "genesis_time": "2023-01-03T18:56:26.31098759Z",
+  "genesis_time": "2023-01-06T14:10:51.279209Z",
   "initial_height": "0",
   "validators": [
```

Comment:
@daniel1302 are these the validator identities we want here? I guess yes, if that's what the mainnet mirror is running with but no if it's the mainnet identities.

Reply:
This is the mainnet mirror. We will have to update mainnet genesis with similar PR, here: https://github.com/vegaprotocol/networks/blob/master/mainnet1/genesis.json